### PR TITLE
Start batch jobs with the correct presubmits for the master branch.

### PR DESCRIPTION
**Pull request by rmmh**
Fixes kubernetes/kubernetes#53923.

### CODE REVIEW

File: prow/cmd/splice/main.go [hunk 1]
```diff
@@ -237,6 +237,9 @@ func requiredPresubmits(presubmits []config.Presubmit) []config.Presubmit {
 		if job.SkipReport { // Ignore silent jobs as these do not block
 			continue
 		}
+		if !job.RunsAgainstBranch("master") { // Ignore jobs that don't run on master
```
Comment (ixdy) [inline prow/cmd/splice/main.go]: splice only works on master right now?

Comment (rmmh) [reply prow/cmd/splice/main.go]: Yes. Tide works on all branches, but splice only bundles PRs targeting master.

**Comment by kargakis** [comment]
/lgtm

Needs a rebase though

**Comment by rmmh** [comment]
Deployed.

**Pull request merged**